skyportal/handlers/api/gcn.py

```diff
@@ -0,0 +1,157 @@
+# Inspired by https://github.com/growth-astro/growth-too-marshal/blob/main/growth/too/gcn.py
+
+import datetime
+import os
+import gcn
+import lxml
+import xmlschema
+from urllib.parse import urlparse
+
+from baselayer.app.env import load_env
+
+from sqlalchemy.orm.exc import NoResultFound
+
+from baselayer.app.access import auth_or_token
+from ..base import BaseHandler
+from ...models import (
+    DBSession,
+    GcnEvent,
+    GcnNotice,
+    GcnTag,
+    Localization,
+)
+from ...utils.gcn import get_dateobs, get_tags, get_skymap, get_contour
+
+gcn.include_notice_types(
```

Comment:
This does nothing by itself. gcn.include_notice_types() is a decorator. If you want to separate the list of notice types from the handler function, you need to assign this to something and use the variable later as a decorator.

Reply:
Yes good point. This can just be removed as we do not setup the cron ingestion in this PR.

Reply:
Well, I _hope_ that you are not going to use cron. I _hope_ that you are going to receive push notifications.

Reply:
Yes total nonsense. By cron, I somehow mean the process by which we will receive notifications.

Reply:
If I understand correctly, the issue is simply that you have a generic BaseLayer infrastructure for event handling, and you want want to keep the GCN notifications within that framework? So all you need to do is write a trivial handler function here that pumps the event to your BaseLayer handler. Why not do that in this PR?

Reply:
I think any persistent service (like those used to received notifications) will need to go in skyportal/services under supervisor... @kmshin1397? I was just thinking we decouple that service from this PR.

Reply:
:+1: I understand now.

Reply:
Yup, exactly. That sounds good @mcoughlin